lib/streamlit/elements/widgets/text_widgets.py

```diff
@@ -421,22 +425,23 @@ def text_area(
             See the ``body`` parameter of |st.markdown|_ for additional,
             supported Markdown directives.
 
-            For accessibility reasons, you should never set an empty label (label="")
-            but hide it with label_visibility if needed. In the future, we may disallow
-            empty labels by raising an exception.
+            For accessibility reasons, you should never set an empty label, but
+            you can hide it with ``label_visibility`` if needed. In the future,
+            we may disallow empty labels by raising an exception.
 
             .. |st.markdown| replace:: ``st.markdown``
             .. _st.markdown: https://docs.streamlit.io/develop/api-reference/text/st.markdown
 
-
         value : object or None
             The text value of this widget when it first renders. This will be
             cast to str internally. If ``None``, will initialize empty and
             return ``None`` until the user provides input. Defaults to empty string.
 
         height : int or None
-            Desired height of the UI element expressed in pixels. If None, a
-            default height is used. Height must be at least 68 pixels (4.25rem).
+            Desired height of the UI element expressed in pixels. If this is
+            ``None`` (default), the widget height fits three lines. The height
+            must be at least 68 pixels, which fits two lines. For any value of
```

Comment:
> For any value of ``height``, the user can resize the widget within their session.

I personally would leave this sentence out as I initially was confused by what this means. I think a developer who uses such a widget sees the resize corner of the box, but I would not put focus on this in the docs (but just my opinion 🙂)

Reply:
How about:
```
            Desired height of the UI element expressed in pixels. If this is
            ``None`` (default), the widget's initial height fits three lines. The height
            must be at least 68 pixels, which fits two lines.
```